html/js/tree/Node.js

```diff
@@ -0,0 +1,94 @@
+GAMBIT.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new Node.
+    * @class
+    * @param {Node} [father] Father node. If null, this is root.
+    */
+    function Node(father) {
+        this.father = father;
+        this.children = [];
+
+        if (father === null) { // If this is root set level to 0
+            this.level = 0;
+        } else {
+            father.addChild(this);
+            this.level = father.level + 1;
+        }
+    }
+
+    /**
+    * Function that draws the node in the global canvas
+    */
+    Node.prototype.draw = function () {
+        var thisNode = this;
+        var circle = GAMBIT.canvas.circle(GAMBIT.CONSTANTS.CIRCLE_SIZE)
+            .y(this.y)
+            .x(this.x)
+            .mouseover(function() {
+                this.fill({ color: '#f06' });
+            })
+            .mouseout(function() {
+                this.fill({ color: '#000' });
+            })
```

Comment:
Instead of mouseover and mouseout, I think it would be better to use CSS classes, to separate code from style (and also make it more efficient as we save two listeners per node). 

You can just add a class .node to all nodes, and then create rules like:

```css
.node {
  fill: #f06;
}
.node:hover {
  fill: #000;
}
```

If we do this, I'd suggest creating a new CSS file only for svg related elements, isntead of reusing the existing main.css